docs: Analytics addition to readme (#987)
Flank/flank: docs/readme.md

- `gradle build`

## Analytics

> Flank makes use of various analytics. For more information on what they are and how to control them please see [analyitics.md](analytics.md)

Reviewer: I think that we should be consistent here, later on, we said that only Bugsnag is used (not various analytics)
```suggestion
Flank makes use of Bugsnag for analytics. For more information about analytics and how to control them please see [analyitics.md](analytics.md)
```

What do you think?

Author: Agreed. Thanks @piotradamczyk5. I thought it used various analytics at first beyond Bugsnag. But @bootstraponline cleared that it was only Bugsnag and i did not clean this definition up
